langchain/src/document_loaders/web/notionapi.ts

```diff
@@ -1,11 +1,20 @@
-import { Client, isFullPage, iteratePaginatedAPI } from "@notionhq/client";
 import {
-  GetPageResponse,
+  Client,
+  isFullBlock,
+  isFullPage,
+  iteratePaginatedAPI,
+} from "@notionhq/client";
+import {
```

Comment:
See below.

Reply:
I did manage to infer the types from the TypeGuards provided by the `@notionhq/client`.